### PR TITLE
[docs] Enforce to ask more details when reporting a bug report

### PR DESCRIPTION
**Pull request by uilianries**
Frequently we have broken packages reported without any further information, making it impossible to be reproduced. Users should be more inclined to populate useful information.

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — COMMENTED** [review]
Maybe add

```yml
validations:
      required: true
```

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

File: .github/ISSUE_TEMPLATE/package_bug.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ body:
   attributes:
     value: |
       Please don't forget to update the issue title.
-      Include all applicable information to help us reproduce your problem.
+      Missing profile or a short log will make impossible to investigate your case,
```
Comment (prince-chrismc) [inline .github/ISSUE_TEMPLATE/package_bug.yml:10]:
```suggestion
      Missing profile or a short log will make extremely difficult to investigate your case or provide any help,
```

**Comment by uilianries** [comment]
@prince-chrismc I updated all fields to be mandatory. Who wants to skip it, can fill with any information. However, I hope it enforces people to inform useful information instead of only a short information.

**Comment by uilianries** [comment]
@AndreyMlashkin good idea!

**Comment by prince-chrismc** [comment]
TIL - If you view file you can debug them 

image

File: .github/ISSUE_TEMPLATE/package_bug.yml [hunk 2]
```diff
@@ -3,41 +3,71 @@ description: 'Report a bug, something does not work as it supposed to'
 title: '[package] <LIBRARY-NAME>/<LIBRARY-VERSION>: SHORT DESCRIPTION'
 labels: bug
 body:
-- type: markdown
+- type: textarea
   attributes:
+    label: Description
+    description: What is not working
```
Comment (prince-chrismc) [inline .github/ISSUE_TEMPLATE/package_bug.yml:9]:
```suggestion
    description: What is not working? What were you expecting? Are there any workarounds?
```

File: .github/ISSUE_TEMPLATE/package_bug.yml [hunk 3]
```diff
@@ -3,41 +3,71 @@ description: 'Report a bug, something does not work as it supposed to'
...
     value: |
       Please don't forget to update the issue title.
```
Comment (prince-chrismc) [inline .github/ISSUE_TEMPLATE/package_bug.yml:11]:
```suggestion
      _Please don't forget to update the issue title._
      
```

**Comment by uilianries** [comment]
> TIL - If you view file you can debug them
> 
> image

Yes, I did the same, but I only found it because when it was broken, the issue label did not show on /issues/ url

File: .github/ISSUE_TEMPLATE/package_bug.yml [hunk 4]
```diff
@@ -3,41 +3,72 @@ description: 'Report a bug, something does not work as it supposed to'
...
     value: |
      * Package Name/Version: **zlib/1.2.8**
      * Operating System+version: **Linux Ubuntu 18.04**
      * Compiler+version: **GCC 8**
      * Docker image: **conanio/gcc8**
      * Conan version: **conan 1.18.0**
      * Python version: **Python 3.7.4**
+    placeholder: |
+      cat /etc/lsb-release | grep
+      gcc --version
```
Comment (AndreyMlashkin) [inline .github/ISSUE_TEMPLATE/package_bug.yml:31]: What if a different compiler used?
Also, this information is already in a conan profile

**Review by jgsogo — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/package_bug.yml (hunk 4 above)
Comment (prince-chrismc) [reply .github/ISSUE_TEMPLATE/package_bug.yml:31]: This is only shown if they delete all the previous text playing with it now that it's merged

The place holder can not be copy pasted so they'll need to take inspiration 🙌 😀 hopefully

**Pull request merged**